man/man8/zpoolprops.8

```diff
@@ -280,6 +282,13 @@ A text string consisting of printable ASCII characters that will be stored
 such that it is available even if the pool becomes faulted.
 An administrator can provide additional information about a pool using this
 property.
+.It Sy dedup_table_quota Ns = Ns Ar number
+This property sets the quota size for the dedup table for the pool.
```

Comment:
```suggestion
This property sets a limit on the on-disk size of the pool's dedup table.
```
Please also add an explanation of the intended usage with special/dedup vdevs, and the relationship to in-memory usage.

Reply:
I'm not quite sure how I would phrase "this applies to DDTs on normal vdevs and special/dedup vdevs."  As for the in-memory portion... that's another one I'm not really sure how to phrase -- it's the same relationship that currently exists, except that the on-disk table gets capped.  Hm.  I can note that the table size may exceed the quota by an amount, based on the write patterns to the pool.